<launch>
  <node name="urg_node" pkg="urg_node" type="urg_node">
    <param name="ip_address" value=""/>
    <param name="serial_port" value="/dev/ttyACM0"/>
    <param name="serial_baud" value="115200"/>
    <param name="frame_id" value="base_laser"/>
    <param name="calibrate_time" value="false"/>
    <param name="publish_intensity" value="false"/>
    <param name="publish_multiecho" value="false"/>
    <param name="angle_min" value="-2.0943951"/>
    <param name="angle_max" value="2.0943951"/>
  </node>
</launch>
